Description: Bump the minimum compatibility level to C++20
 This pull request is very similar to
 https://github.com/lxqt/lxqt-build-tools/pull/83, except this is proactive
 rather than reactive.
Author: Simon Quigley <tsimonq2@ubuntu.com>
Origin: upstream
Forwarded: https://github.com/lxqt/lxqt-build-tools/pull/102
Last-Update: 2025-01-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cmake/modules/LXQtCompilerSettings.cmake
+++ b/cmake/modules/LXQtCompilerSettings.cmake
@@ -181,11 +181,11 @@ endif()
 
 
 #-----------------------------------------------------------------------------
-# CXX17 requirements - no checks, we just set it
+# CXX20 requirements - no checks, we just set it
 #-----------------------------------------------------------------------------
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ ISO Standard")
+set(CMAKE_CXX_STANDARD 20 CACHE STRING "C++ ISO Standard")
 
 
 #-----------------------------------------------------------------------------
